src/formatters/breadcrumbs.coffee

```diff
@@ -4,7 +4,7 @@
 # Set defaults and cast vars
 url = url || process.env.URL
 
-export default (breadcrumbsList = []) ->
+export default ({ breadcrumbsList = [], name }) ->
```

Comment:
Ok, then how about we set a default, since I _doubt_ we'll remember to set this on projects since it's so nebulous.  Like:

```suggestion
export default ({ breadcrumbsList = [], name = 'Breadcrumbs' }) ->
```

Reply:
Yeah, that's a great idea!